computation-slurm/src/test/resources/expectedShell/argsWithSpaces.batch

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+case $SLURM_ARRAY_TASK_ID in
+    0)
+      ARGS=""line\ 1,line\ 2" "v2""
```

Comment:
I am not sure having quotes into quotes is a good idea (I am pretty surethe second quote closes the first) : maybe we can remove the inner quotes, in the generated args, since now we escape characters ?